### ISSUE TITLE
Support for multi page documentation

### ISSUE THREAD

**Issue by rwinch** [asciidoctor/asciidoctor-gradle-plugin#52]
I'm also interested in this feature and am willing to contribute support for it. For the initial discussion see http://discuss.asciidoctor.org/Recommendations-for-multi-page-and-single-page-documentation-td118.html

**Comment by gAmUssA** [comment]
@rwinch Any progress in this?

**Comment by ysb33r** [comment]
@mojavelinux Has there been any chancing in asciidoctor core to be able to support this?

**Comment by mojavelinux** [comment]
No. I'm currently investing it another strategy, which is to pre-chunk the content and recombine as needed.

The experimentation for multi-page converter has been happening here: https://github.com/asciidoctor/asciidoctor-extensions-lab/pull/96 Perhaps check with @owenh000. I'm happy to merge in his changes, I just haven't gotten around to it yet.

**Comment by ysb33r** [comment]
I am going to close this as it really is out of scoipe of the Gradle plugin.
